### PR TITLE
Create 04-mechanics-of-contributing-article-es.asciidoc

### PR DESCRIPTION
**Pull request by rrrutledge**
Translated using DeepL

### CODE REVIEW

File: contributor/es/04-mechanics-of-contributing-article-es.asciidoc
Comment (fioddor) [inline contributor/es/04-mechanics-of-contributing-article-es.asciidoc:148]:
```suggestion
  *** Algunas herramientas también tienen la funcionalidad de marcar el pull request como borrador / WIP que puedes utilizar para explícitar el trabajo inacabado y no pulido y aún así obtener retroalimentación temprana del https://innersourcecommons.org/learn/learning-path/trusted-committer/02/[_Trusted Committer_] de tu equipo anfitrión.
```

File: contributor/es/04-mechanics-of-contributing-article-es.asciidoc
Comment (fioddor) [inline contributor/es/04-mechanics-of-contributing-article-es.asciidoc:149]:
```suggestion
  *** Esto te permite asegurarte de que vas por un camino que tu equipo anfitrión estará feliz de fusionar una vez que esté hecho, adhiriéndote en cierto modo a la idea de "liberar temprano, liberar a menudo".
```

**Review by fioddor — CHANGES_REQUESTED** [review]
I think we'd benefit from a second pass with a fresh mind (mine is mudded, right now)

**Comment by rrrutledge** [comment]
> I think we'd benefit from a second pass with a fresh mind (mine is mudded, right now)

Sure!  @Sergih28 has contributed in the past and may be interested?

**Comment by rrrutledge** [comment]
> I think we'd benefit from a second pass with a fresh mind (mine is mudded, right now)

We've had @JFernando122 review some in the past who may be willing to take a look?

**Comment by Sergih28** [comment]
> > I think we'd benefit from a second pass with a fresh mind (mine is mudded, right now)
> 
> Sure! @Sergih28 has contributed in the past and may be interested?

I can probably take a look later on this week.

**Comment by rrrutledge** [comment]
> I can probably take a look later on this week.
Oh nice - thank you! 🎉

**Comment by rrrutledge** [comment]
> I can probably take a look later on this week.

Thanks!  When you've had a look just tick the `Approve` button (or leave comments if you have any).

**Comment by Sergih28** [comment]
> Thanks! When you've had a look just tick the `Approve` button (or leave comments if you have any).

I'll set up a reminder to take a look this week, busy days sorry :/

**Comment by rrrutledge** [comment]
> busy days sorry

No problem!  You’re doing great!

File: contributor/es/04-mechanics-of-contributing-article-es.asciidoc
Comment (JFernando122) [inline contributor/es/04-mechanics-of-contributing-article-es.asciidoc:4]: I think the original one ' Do you look forward to reducing your blockers not by management escalation but by collaboration?' means more like reduce the blockers by collaborating (doing it yourself) instead than waiting for other person to do it

File: contributor/es/04-mechanics-of-contributing-article-es.asciidoc
Comment (JFernando122) [inline contributor/es/04-mechanics-of-contributing-article-es.asciidoc:5]: 'Tengan' is more Spanish neutral imo
```suggestion
En esta sección se ofrecen consejos prácticos y se destacan los puntos que hay que recordar al realizar una contribución en InnerSource. Permite que el equipo anfitrión y tú tengan una experiencia lo más agradable posible, sentando las bases para más contribuciones y una gran colaboración.
```

File: contributor/es/04-mechanics-of-contributing-article-es.asciidoc
Comment (JFernando122) [reply contributor/es/04-mechanics-of-contributing-article-es.asciidoc:13]:
```suggestion
Si tu contribución es más grande, posiblemente pasarás por (algunos) de estos pasos repetidamente mientras iteras hacia el objetivo común.
```
saying 'tu' is weird if it is a common objective cause it makes it look like is a personal objective, 'el' makes it so the objective is shared between all parts

File: contributor/es/04-mechanics-of-contributing-article-es.asciidoc
Comment (JFernando122) [inline contributor/es/04-mechanics-of-contributing-article-es.asciidoc:32]: This is a link to a resource down below and it needs to have the same name to link properly
```suggestion
Este efecto puede observarse también en el código abierto, puede leer más sobre ello <<Creación de confianza a través de la colaboración, aquí>>.
```

File: contributor/es/04-mechanics-of-contributing-article-es.asciidoc
Comment (JFernando122) [reply contributor/es/04-mechanics-of-contributing-article-es.asciidoc:39]: Also missing the article for trusted committer ```suggestion
Un enfoque es reaccionar rápidamente con un "lo miraré, aunque no lo haré en los próximos días" a un comentario de un https://innersourcecommons.org/learn/learning-path/trusted-committer[_Trusted Committer_] si sabes que sólo podrás atenderles dentro de unos días.
```

File: contributor/es/04-mechanics-of-contributing-article-es.asciidoc
Comment (JFernando122) [inline contributor/es/04-mechanics-of-contributing-article-es.asciidoc:77]: Trailing space, i also noted that it is in the original version
```suggestion
En primer lugar, estarás en contacto menos directo con ellos. En segundo lugar, no se espera que estés tan bien informado y seas tan competente como en los proyectos a tiempo completo que posee tu equipo.
```

File: contributor/es/04-mechanics-of-contributing-article-es.asciidoc
Comment (JFernando122) [inline contributor/es/04-mechanics-of-contributing-article-es.asciidoc:87]:
```suggestion
A veces el status quo surge por pura casualidad y se mantiene así porque nadie tuvo una idea diferente o se preocupó lo suficiente.
```

File: contributor/es/04-mechanics-of-contributing-article-es.asciidoc
Comment (JFernando122) [inline contributor/es/04-mechanics-of-contributing-article-es.asciidoc:104]: I feel like [indentación](https://es.wikipedia.org/wiki/Indentación) is more used in programming even tho its not recognized as a word according to wikipedia 🤔
```suggestion
Todos tenemos nuestras preferencias y opiniones sobre el estilo del código, la indentación, etc.
```

File: contributor/es/04-mechanics-of-contributing-article-es.asciidoc
Comment (JFernando122) [inline contributor/es/04-mechanics-of-contributing-article-es.asciidoc:119]:
```suggestion
Imagina que has construido una característica o el manejo de una rareza irresoluble, o un importante ajuste de rendimiento, y tu código no es del todo obvio (o incluso podría parecer tramposo / incorrecto a primera vista).
```

File: contributor/es/04-mechanics-of-contributing-article-es.asciidoc
Comment (JFernando122) [inline contributor/es/04-mechanics-of-contributing-article-es.asciidoc:113]: I saw down below the suggestion of translating ´merge´ as 'integración' instead of 'fusión' and i think it fits better
```suggestion
Has completado todo el trabajo esencial, has descubierto todas las peculiaridades del problema y del proyecto al que estás contribuyendo, se acerca el momento en el que has planeado que se utilice la nueva característica y quieres asegurarte de que tu contribución se integre de la forma más rápida y fluida posible.

Esto es lo que puedes hacer para que la revisión y la integración sean lo más fácil posible para el https://innersourcecommons.org/learn/learning-path/trusted-committer[_Trusted Committer_] y el equipo anfitrión.
```

File: contributor/es/04-mechanics-of-contributing-article-es.asciidoc
Comment (JFernando122) [inline contributor/es/04-mechanics-of-contributing-article-es.asciidoc:137]: Feels weird to not have the link as it always have it, also its missing in the original version
```suggestion
Para que la revisión del código sea lo más fácil posible para el https://innersourcecommons.org/learn/learning-path/trusted-committer[_Trusted Committer_] u otras personas que lo revisen, intenta seguir estos consejos:
```

**Review by JFernando122 — CHANGES_REQUESTED** [review]
Long time no see, i finally got some free time to spare and made some reviews but i see fioddor already gave some good feedback

File: contributor/es/04-mechanics-of-contributing-article-es.asciidoc
Comment (fioddor) [reply contributor/es/04-mechanics-of-contributing-article-es.asciidoc:4]: Any suggestion?

File: contributor/es/04-mechanics-of-contributing-article-es.asciidoc
Comment (fioddor) [reply contributor/es/04-mechanics-of-contributing-article-es.asciidoc:5]: We don't seek neutrality (distance to the reader) and _tengan_ would be incoherent with the casual form _tú_.

File: contributor/es/04-mechanics-of-contributing-article-es.asciidoc
Comment (fioddor) [reply contributor/es/04-mechanics-of-contributing-article-es.asciidoc:119]: Nice catch. We indeed must avoid _hacky_.
I don't like _tramposo_, though. _truculento_, perhaps?

File: contributor/es/04-mechanics-of-contributing-article-es.asciidoc
Comment (Sergih28) [reply contributor/es/04-mechanics-of-contributing-article-es.asciidoc:5]: Isn't `tengáis` the correct spelling?

File: contributor/es/04-mechanics-of-contributing-article-es.asciidoc
Comment (Sergih28) [reply contributor/es/04-mechanics-of-contributing-article-es.asciidoc:104]: I agree, I've always used `indentación` in terms of programming.

I wouldn't feel comfortable adding a _non-existent_ word, but in this case, given that it has an article on Wikipedia, I suppose it could work?

File: contributor/es/04-mechanics-of-contributing-article-es.asciidoc
Comment (Sergih28) [reply contributor/es/04-mechanics-of-contributing-article-es.asciidoc:119]: Coul `lioso` fit here?

File: contributor/es/04-mechanics-of-contributing-article-es.asciidoc
Comment (Sergih28) [reply contributor/es/04-mechanics-of-contributing-article-es.asciidoc:4]: I can't think of a good translation, but I'll leave this as an idea / hint:
`reducir esos momentos en que te encuentras atascado`

File: contributor/es/04-mechanics-of-contributing-article-es.asciidoc
Comment (JFernando122) [reply contributor/es/04-mechanics-of-contributing-article-es.asciidoc:119]: i think truculoso fits better

File: contributor/es/04-mechanics-of-contributing-article-es.asciidoc
Comment (JFernando122) [reply contributor/es/04-mechanics-of-contributing-article-es.asciidoc:104]: Yeah i mean most videos i have watched they say indentacion, and also at college most teachers/students said it. And the article itself says its used in the programming community so i dont see why we shouldnt use it

File: contributor/es/04-mechanics-of-contributing-article-es.asciidoc
Comment (JFernando122) [reply contributor/es/04-mechanics-of-contributing-article-es.asciidoc:5]: tengais is used in some parts. Also i suggested tengan because it refers to both the host team and the reader, so its plural.

File: contributor/es/04-mechanics-of-contributing-article-es.asciidoc
Comment (JFernando122) [reply contributor/es/04-mechanics-of-contributing-article-es.asciidoc:4]: Cant think of a good translation neither, this is the best i have
`¿Quieres eliminar tus bloqueos en lugar de esperar a que alguien mas lo haga?`

File: contributor/es/04-mechanics-of-contributing-article-es.asciidoc
Comment (fioddor) [reply contributor/es/04-mechanics-of-contributing-article-es.asciidoc:5]: _tengáis_ is plural as well. It's the casual form instead of the formal one.

**Comment by rrrutledge** [comment]
@fioddor would you be able to resolve the remaining comments and merge?  Looking forward to this!

File: contributor/es/04-mechanics-of-contributing-article-es.asciidoc
Comment (Sergih28) [reply contributor/es/04-mechanics-of-contributing-article-es.asciidoc:119]: I've heard the word _enrevesado_ but never heard about the word _truculento_.
That's all I can contribute here :sweat_smile:

**Pull request merged**
